### DISCUSSION TITLE
Insights Panel: Full-Screen Preview

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#16543] [Feature Requests]
It would be nice to have a small icon on panel hover (or a right-click menu) that allows you to "zoom" that panel into a temporary full-screen so you can see all the details more eaasily.

**Comment by benhaynes** [comment]
Once open, you should be able to use the arrow keys to navigate through the panels while staying in full-screen

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
